api/server/daemon.go

```diff
@@ -178,3 +181,23 @@ func (s *Server) getEvents(ctx context.Context, w http.ResponseWriter, r *http.R
 		}
 	}
 }
+
+func getLabels(d *daemon.Daemon, id string) map[string]string {
+	container, err := d.Get(id)
```

Comment:
This is going to be a no-go since we are making the effort to move any such interactions into the daemon itself rather than having in it in the api layer.
Maybe we can move the majority of the things in the `getEvents` function into the daemon, and this can go along with it?